### PR TITLE
Feat/forecast and notifications

### PR DESCRIPTION
**Pull request by austinbrown-opspark**

### CODE REVIEW

File: client/components/Itinerary.js [hunk 1]
```diff
@@ -168,6 +141,7 @@ const Itinerary = ({ currentUser, currentTrip, day }) => {
           setActivities(newActivities);
         }}
       />
+      <br />
```
Comment (jstrzesz) [inline client/components/Itinerary.js:145]: use css to format spacing in components, not br tags

File: client/components/ResponsiveDrawer.js [hunk 2]
```diff
@@ -333,6 +349,8 @@ const ResponsiveDrawer = ({ currentUser, currentTrip }) => {
         currentUser={currentUser}
         setClickedPage={setClickedPage}
       />
+      <br />
```
Comment (jstrzesz) [inline client/components/ResponsiveDrawer.js:346]: see above comment about br tags

File: client/components/Itinerary.js (hunk 1 above)
Comment (austinbrown-opspark) [reply client/components/Itinerary.js]: fixed

File: client/components/ResponsiveDrawer.js (hunk 2 above)
Comment (austinbrown-opspark) [reply client/components/ResponsiveDrawer.js]: fixed

**Review by joshjnunez — APPROVED** [review]

**Review by gjones2019 — APPROVED** [review]

**Pull request merged**